lib/services/create_basket_item.rb

```diff
@@ -1,15 +1,23 @@
 module Shop
   class CreateBasketItem
+    attr_reader :product_id
+    attr_accessor :quantity
+
     def initialize(params)
       @product_id = params.fetch("product_id").to_i
       @quantity = params.fetch("quantity").to_i
     end
 
     def call
-      BASKET_ITEMS << BasketItem.new(
-        product_id: @product_id,
-        quantity: @quantity
-      )
+      item_in_basket = BASKET_ITEMS.find { |item| item.product_id == @product_id }
```

Comment:
Thanks for suggestions. I changed the code locally. Since I merged my PR  (per @gotar 's advice) before your comments, I'm not sure how to proceed. 
Should I just push these new changes to master now, or just wait for my next PR?